core/src/main/java/io/kestra/core/models/tasks/FlowableTask.java

```diff
@@ -73,7 +73,7 @@ default Optional<State.Type> resolveState(RunContext runContext, Execution execu
         );
     }
 
-    default T outputs(RunContext runContext, Execution execution, TaskRun parentTaskRun) throws IllegalVariableEvaluationException {
+    default T outputs(RunContext runContext) throws Exception {
```

Comment:
Can we only throws IllegalVariableEvaluationException as before?

Reply:
The outputs implementation for WorkingDirectory can throw an IOException and it's better to propagate the exception.